pkgs/development/tools/rgp/default.nix

```diff
@@ -13,16 +13,18 @@
 , libXi
 , ncurses5
 , qtbase
+, qtdeclarative
 , zlib
 }:
 
 stdenv.mkDerivation rec {
   pname = "rgp";
-  version = "1.7.0.29";
+  version = "1.8";
+  buildNum = "2020-09-11-26";
```

Comment:
Could you make this a `let` binding? Helps keeping the environment build environment clean.